pyrk/inp/sim_info.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
         self.rho_ext = self.init_rho_ext(rho_ext)
         self.feedback = feedback
         self.ne = self.init_ne()
-        self.th = th_system.THSystem(kappa=kappa, components=components)
+        self.th = th_system.THSystemSphFVM(kappa=kappa, components=components)
```

Comment:
As per my earlier comment, this needs to be fixed. Since not all systems are THSystemSphFVM, it doesn't make sense to make them objects of that class.